src/database/seeders/20200109111618-demo-role.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert(
+      'Roles',
+      [
+        {
+          id: 'e4af43af-ca38-426b-a8ec-65a52a5a45fc',
+          name: 'Admin',
+          description: 'The boss himself',
+        },
+      ],
+      {},
+    );
+  },
```

Comment:
Can you add more roles such as the `Owner`, `Volunteer`, `Super Admin`, and `Manager`?

Reply:
Sure